source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Domain/Dtos/ChargeCommands/Validation/InputValidation/Factories/ChargeOperationInputValidationRulesFactory.cs

```diff
@@ -97,6 +97,7 @@ private List<IValidationRuleContainer> CreateRulesForChargeInformation(ChargeOpe
                 CreateRuleContainer(new ChargeDescriptionRequiredRule(chargeOperationDto), chargeOperationDto.Id),
                 CreateRuleContainer(new ResolutionIsRequiredRule(chargeOperationDto), chargeOperationDto.Id),
                 CreateRuleContainer(new TransparentInvoicingValidationRule(chargeOperationDto), chargeOperationDto.Id),
+                CreateRuleContainer(new TaxIndicatorValidationRule(chargeOperationDto), chargeOperationDto.Id),
```

Comment:
Should we do this, like `ResolutionIsRequiredRule`:

```suggestion
                CreateRuleContainer(new TransparentInvoicingIsRequiredValidationRule(chargeOperationDto), chargeOperationDto.Id),
                CreateRuleContainer(new TaxIndicatorIsRequiredValidationRule(chargeOperationDto), chargeOperationDto.Id),
```

Also, it appears that some of our rules are called "ValidationRule" while other are just "Rule". Maybe we should align, perhaps in another PR.

Reply:
done